stdlib/2/itertools.pyi

```diff
@@ -119,8 +119,9 @@ def izip(iter1: Iterable[Any], iter2: Iterable[Any],
 def izip_longest(*p: Iterable[Any],
                  fillvalue: Any = ...) -> Iterator[Any]: ...
 
+# Mypy thinks this overlaps with the last overload, but produces correct types anyway.
```

Comment:
Let me double check that it still thinks that after the recent overload revamps.

Reply:
Indeed, both `# type: ignore`s are no longer needed. I'll get rid of them.